### DISCUSSION TITLE
Add event

### DISCUSSION THREAD

**Discussion by ksiegel1923** [vkurko/calendar#232] [Q&A]
Hi I am having trouble adding an event once I finish dragging on the calendar. I figured out how to use the select callback so that the code is aware that I have made a selection but when I try to call the addEvent function it says that it doesn't exist. Does anyone have an example of doing this?

**Comment by vkurko** [comment]
I have prepared a simple [demo](https://jsbin.com/tuqugeveve/edit?html,output) , where the selected period is immediately added to the list of events in the calendar.
